packages/flutter_tools/lib/src/web/file_generators/wasm_bootstrap.dart

```diff
@@ -18,6 +20,7 @@ String generateWasmBootstrapFile(bool isSkwasm) {
     moduleInstance = await dart2wasm_runtime.instantiate(dartModulePromise, imports);
   } catch (exception) {
     console.error(`Failed to fetch and instantiate wasm module: \${exception}`);
+    console.error($kWasmMoreInfo);
```

Comment:
you should have a test for the bug here, at least. :-)

Reply:
The bug is running a browser without the right flags. I can add a TODO for @eyebrowsoffire

Reply:
Oh 🤦